.prettierrc.json

```diff
@@ -1,5 +1,4 @@
 {
   "printWidth": 120,
-  "parser": "typescript",
```

Comment:
I'm concerned about removing this!

Reply:
According to `prettier` (emphasis is mine):
> Prettier automatically infers the parser from the input file path, so **you shouldn't** have to change this setting [1]

[1] https://prettier.io/docs/en/options.html#parser

Reply:
Hmm... Did you try it? It's simple: if it works when you run both tslint and prettier, I'm fine with that.
I introduced this change because I had conflicts between prettier and tslint about semi-colons: the former was adding semi-colon, the latter was removing them.
Maybe is it because of .tsx files and not .ts ?

Reply:
> I introduced this change because I had conflicts between prettier and tslint about semi-colons: the former was adding semi-colon, the latter was removing them.
Maybe is it because of .tsx files and not .ts ?

Not sure about that, `ts-lint` is configured [1] to add semicolons:
```json
"semicolon": [true, "always"],
```
and `prettier` [2] default is `true`

`npm run prettier` and `npm run build` runs fine, no file is modified and there is no trace of errors. 

[1] https://github.com/swift-sunshine/swsui/blob/master/tslint.json#L66
[2] https://prettier.io/docs/en/options.html#semicolons

Reply:
Also travis is running it too without fails (no file needs to be modified) https://travis-ci.org/swift-sunshine/swsui/builds/349408486#L3781